artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/management/ActiveMQServerControl.java

```diff
@@ -521,6 +521,9 @@ public interface ActiveMQServerControl {
    @Attribute(desc = "The runtime size of the authorization cache")
    long getAuthorizationCacheSize();
 
+   @Attribute(desc = "The current status of the server")
+   String getStatus();
+
```

Comment:
Having a 'status' attribute ("The current status of the server") which isnt actually a server status but just a configurable config string that defaults to empty, seems off to me. I would call this something else either more specific or more general. Seems more like the intent is to use it as a config update correlation of sorts, perhaps wind that function into the name?

Reply:
the other thought, getStatusJson - but I think that is over kill, we have xxJson methods that are better than the xx, so for a new one, we can just return a string of json or not with out the method postfix

Reply:
I dont think 'status' as an attribute is that general, it implies - and actually outright states for its description here - that it returns the status of the server, which I wouldnt really say it does in this case.

I think if the value is expected to be in a certain format later it should adhere to that from the start...this is described on the JIRA and exposed as being a freeform string set by the config, so 'other things adding keys later' doenst seem to work / is effectively breaking.

Reply:
maybe I need to state that it will be json, or empty, but the attributes are up for grabs. At the moment it is only settable by config so there is no coordination, no format.